Pod/Classes/NWSTokenView.swift

```diff
@@ -47,8 +40,33 @@ public class NWSTokenView: UIView, UIScrollViewDelegate, UITextViewDelegate
     
     // MARK: Private Vars
     private var shouldBecomeFirstResponder: Bool = false
-    private var scrollView = UIScrollView()
+    public var scrollView = UIScrollView()
     public var textView = UITextView()
+    public var toTextColor = UIColor.blackColor() {
```

Comment:
@wfilleman I would rename this to 'tokenViewLabelTextColor' as 'to' is only a possible value users can use.